Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -93,6 +97,54 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
         }
     }
 
+    private func createEmptyStateOverlayView() -> UIView {
+        let overlayView = UIView()
+        overlayView.backgroundColor = UIColor.whiteColor()
+
+        let logoImageView = UIImageView(image: UIImage(named: "emptyBookmarks"))
+        overlayView.addSubview(logoImageView)
+        logoImageView.snp_makeConstraints { make in
+            make.centerX.equalTo(overlayView)
+
+            // Sets proper top constraint for iPhone 6 in portait and for iPad.
+            make.centerY.equalTo(overlayView.snp_centerY).offset(HomePanelUX.EmptyTabContentOffset).priorityMedium()
+
+            // Sets proper top constraint for iPhone 4, 5 in portrait.
+            make.top.greaterThanOrEqualTo(overlayView.snp_top).offset(50).priorityHigh()
```

Comment:
I'm not sure why this is needed or how this works since the centerY and top constraints would conflict. Since the priority on the top is always higher I don't see the centerY constraint ever holding true. Could we remove this top constraint line and the priorities?